pytorch_toolkit/ote/tests/init_venv.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
```

Comment:
what is it used for?

Reply:
let's require `ote` to be installed before running tests and avoid using sys.path.append. Why do you need openvino here?

Reply:
I'm not sure that it is a good decision. 
At the moment my script `run_model_templates_tests2.py` has option `--list` to print lists of tests (and you can add some filter options and see how they affect on the lists of tests).
I would like to have possibility to print list of tests from a "clean" environment without installed `ote`. 
Now you can print lists of tests, -- it is sufficient to have `pyyaml`, `mmcv` and `texttable` installed in the current environment.

Reply:
@LeonidBeynenson , it's strange to test `ote` without `ote` installed. It is still lightweight and I hope it will be lightweight in future. Please install `ote`

Reply:
I agree that `ote` is lightweight.
But I supposed that we developing `ote` at the moment, so it is better not depend on it

Please, note that the most popular mistake during developing a package is to install it to the current venv.
Very often people install it (without `develop` option!), change something in the package, then run a script depending on the package, and then they will have a strange error, since the changes are not present in the installed package.

As far as I remember, even installing the package in develop mode has its drawbacks -- if we use sometimes absolute import (e.g. `from ote.utils import something`) and sometimes relative import (`from .utils import something`) -- they will have different result if we check several different versions of `ote`.

I agree that the cases that I mention are not often. But nevertheless, I feel that installing `ote` package when we developing it is a fragile decision.

Reply:
I believe you like arguing but, how developer install package during development is a problem of developer.

Please install `ote` with its dependencies